Components/Avatar/Avatar.css

```diff
@@ -1,27 +1,34 @@
-.avatar-md{
+
```

Comment:
fix this problem a

Reply:
actually the comments in CSS file is only for documentation [age and that is not the part of component library and the css file is big so for readability purpose I have added the code

Reply:
Actually, instead of dev I have gone with the main branch and created a branch for avatar and branch changes but then went ahead with it as all components needed for PO and further, I will add P1 components and create a branch from the main